### PR TITLE
Seo Update about

### PR DESCRIPTION
**Pull request by rraue**
https://github.com/mozilla-iot/mozilla-iot.github.io/issues/49

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -26,7 +27,7 @@
     <div class="mzp-c-navigation">
       <div class="mzp-c-navigation-l-content">
         <div class="mzp-c-navigation-container">
-          <div class="mzp-c-navigation-logo"><a id="wordmark" href="/">Mozilla</a></div>
+          <div class="mzp-c-navigation-logo"><a id="wordmark" href="https:ios.mozilla.org">Mozilla</a></div>
```
Comment (mrstegeman) [inline about/index.html]: This link is wrong

Comment (rraue) [reply about/index.html]: Doh, I forgot the //, but in general its better if this is not relative, so every user and crawler at the end gets to iot.mozilla.org and not the duplicate content on x.github.io

```suggestion
          <div class="mzp-c-navigation-logo"><a id="wordmark" href="https://iot.mozilla.org">Mozilla</a></div>
```

Comment (mrstegeman) [reply about/index.html]: That change looks good.

**Comment by mrstegeman** [comment]
Moved to #68

**Pull request closed without merging**